### ISSUE TITLE
API文档之间冲突

### ISSUE THREAD

**Issue by ZhangzrJerry** [PaddlePaddle/Paddle#45071]
### bug描述 Describe the Bug

这是一段最简单的代码，用于多层感知机的搭建
```python
# MLP模型组网搭建
from paddle import nn
lenet_Sequential = nn.Sequential(
    nn.Linear(n_input, 1,),
    nn.Tanh(),
    nn.Linear(1, 20),
    nn.Tanh(),
)
paddle.device.set_device('gpu:0')
# 封装模型为一个 model 实例，便于进行后续的训练、评估和推理
feature = paddle.static.InputSpec([30], dtype='float32', name='feature')
label = paddle.static.InputSpec([1], dtype='int8', name='label')
model = paddle.Model(lenet_Sequential(),inputs=feature,labels=label)
# 为模型训练做准备，设置优化器及其学习率，并将网络的参数传入优化器，设置损失函数和精度计算方式
model.prepare(optimizer=paddle.optimizer.Adam(learning_rate=0.001, parameters=model.parameters()), 
              loss=paddle.nn.L1Loss(), 
              metrics=paddle.metric.Accuracy())
# 启动模型训练，指定训练数据集，设置训练轮次，设置每次数据集计算的批次大小，设置日志格式
model.fit(train_dataset, 
          epochs=1, 
          batch_size=1,
          verbose=2)
```
其中在代码第13行中，修改为`model = paddle.Model(lenet_Sequential(),inputs=feature,labels=label)`报**不同**的错误

在API文档1中https://www.paddlepaddle.org.cn/documentation/docs/zh/guides/beginner/train_eval_predict_cn.html#paddle-model ，二、2.1中传入的模型参数未带括号，
```python
# 封装模型为一个 model 实例，便于进行后续的训练、评估和推理
model = paddle.Model(mnist)
```
在API文档2中https://www.paddlepaddle.org.cn/documentation/docs/zh/practices/quick_start/linear_regression.html ，六、模型训练中，
```model = paddle.Model(UCIHousing())```
，传入的模型参数更像是一个实例化对象，完整的代码见补充信息

### 其他补充信息 Additional Supplementary Information

```python
import pandas as pd
import os
import cv2
import numpy as np
from paddle.io import Dataset

# 导入训练数据
df_train = pd.read_csv("train.csv")
x = np.asarray(df_train.iloc[:,2:]).astype(np.float32)
y = np.array(df_train.iloc[:,1]).astype(np.int8)
# 导入测试数据
df_test = pd.read_csv("test.csv")
x_pred = np.array(df_test.iloc[:,1:])
# 对特征进行归一化
from sklearn.preprocessing import StandardScaler  
scaler = StandardScaler()  
scaler.fit(x)  
x = scaler.transform(x)
x_pred = scaler.transform(x_pred)
# 将训练数据集和测试数据集按照8:2的比例分开
ratio = 0.8
offset = int(x.shape[0] * ratio)
x_train = x[:offset]
y_train = y[:offset]
x_test = x[offset:]
y_test = y[offset:]

class MyDataset(Dataset):
    """
    步骤一：继承 paddle.io.Dataset 类
    """
    def __init__(self, x, y):
        """
        步骤二：实现 __init__ 函数，初始化数据集，将样本和标签映射到列表中
        """
        super(MyDataset, self).__init__()
        self.data_list = []
        for i,j in zip(x,y):
            self.data_list.append([i,j])

    def __getitem__(self, index):
        """
        步骤三：实现 __getitem__ 函数，定义指定 index 时如何获取数据，并返回单条数据（样本数据、对应的标签）
        """
        feature = self.data_list[index][0]
        label = self.data_list[index][1]
        # 返回图像和对应标签
        return feature, label

    def __len__(self):
        """
        步骤四：实现 __len__ 函数，返回数据集的样本总数
        """
        return len(self.data_list)

train_dataset = MyDataset(x_train,y_train)

n_input = len(x[0])
# MLP模型组网搭建
from paddle import nn
lenet_Sequential = nn.Sequential(
    nn.Linear(n_input, 1,),
    nn.Tanh(),
    nn.Linear(1, 20),
    nn.Tanh(),
)
paddle.device.set_device('gpu:0')
# 封装模型为一个 model 实例，便于进行后续的训练、评估和推理
feature = paddle.static.InputSpec([30], dtype='float32', name='feature')
label = paddle.static.InputSpec([1], dtype='int8', name='label')
model = paddle.Model(lenet_Sequential(),inputs=feature,labels=label)
# 为模型训练做准备，设置优化器及其学习率，并将网络的参数传入优化器，设置损失函数和精度计算方式
model.prepare(optimizer=paddle.optimizer.Adam(learning_rate=0.001, parameters=model.parameters()), 
              loss=paddle.nn.L1Loss(), 
              metrics=paddle.metric.Accuracy())
# 启动模型训练，指定训练数据集，设置训练轮次，设置每次数据集计算的批次大小，设置日志格式
model.fit(train_dataset, 
          epochs=1, 
          batch_size=1,
          verbose=2)
```

**Comment by ZhangzrJerry** [comment]
ISSUE是我阅读不够细致造成的

> https://www.paddlepaddle.org.cn/documentation/docs/zh/api/paddle/Model_cn.html#model
要求传入的是一个实例化对象 <img width="603" alt="image" src="https://user-images.githubusercontent.com/87751816/184134004-b78e9909-bb21-408f-ad8b-50f62399bb95.png">

> https://www.paddlepaddle.org.cn/documentation/docs/zh/guides/beginner/train_eval_predict_cn.html#paddle-model
mnist已经是一个对象了 <img width="592" alt="image" src="https://user-images.githubusercontent.com/87751816/184125341-810e42ae-5637-4b23-837c-fc114e09f060.png">

> https://www.paddlepaddle.org.cn/documentation/docs/zh/practices/quick_start/linear_regression.html
这里传入的是一个类的实例<img width="467" alt="image" src="https://user-images.githubusercontent.com/87751816/184125647-1197167b-480e-4abc-af3a-0ce1bbae0408.png">
